metrics/metrics.go

```diff
@@ -0,0 +1,33 @@
+package metrics
+
+import (
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+// Metrics is the metrics collector
+type Metrics struct {
+	gatherer   prometheus.Gatherer
+	registerer prometheus.Registerer
+}
+
+// newMetrics will create a new Metrics object; this is private to ensure only one, global instance is created
```

Comment:
I think that this kind of comment is unnecessary - for me we should keep the rule from our code standards:
"Write self-documenting code and minimize comments."
I think that this function is self-explanatory enough and doesn't need a comment.

Reply:
I've intentionally added this comment to note why the "constructor" is private and mention the `global` instance. Maybe only the first part `newMetrics will create a new Metrics object;` is unnecessary. What do you think?

Reply:
Okay, I agree we can leave the part after ;

Reply:
Done 👍 - check the new comment.